Sources/GRPC/ConnectionPool/HTTP2Connections.swift

```diff
@@ -27,6 +27,18 @@ internal struct HTTP2Connections {
     return self.connections.count
   }
 
+  internal var idleCount: Int {
+    return self.connections.values.reduce(0) { $0 + ($1.isIdle ? 1 : 0) }
```

Comment:
Minor perf note if you want it: this (and the below) can be `&+` as you cannot possibly overflow.

Reply:
Always happy to take perf notes :)